### ISSUE TITLE
Add bootnodes to chain-spec.

### ISSUE THREAD

**Issue by 5chdn** [poanetwork/poa-chain-spec#39]
You can add boot nodes to the chain spec like that:

	"nodes": [
		"enode://56abaf065581a5985b8c5f4f88bd202526482761ba10be9bfdcd14846dd01f652ec33fde0f8c0fd1db19b59a4c04465681fcef50e11380ca88d25996191c52de@40.71.221.215:30303",
		"enode://d07827483dc47b368eaf88454fb04b41b7452cf454e194e2bd4c14f98a3278fed5d819dbecd0d010407fc7688d941ee1e58d4f9c6354d3da3be92f55c17d7ce3@52.166.117.77:30303",
		"enode://8fa162563a8e5a05eef3e1cd5abc5828c71344f7277bb788a395cce4a0e30baf2b34b92fe0b2dbbba2313ee40236bae2aab3c9811941b9f5a7e8e90aaa27ecba@52.165.239.18:30303",
		"enode://7e2e7f00784f516939f94e22bdc6cf96153603ca2b5df1c7cc0f90a38e7a2f218ffb1c05b156835e8b49086d11fdd1b3e2965be16baa55204167aa9bf536a4d9@52.243.47.56:30303",
		"enode://0518a3d35d4a7b3e8c433e7ffd2355d84a1304ceb5ef349787b556197f0c87fad09daed760635b97d52179d645d3e6d16a37d2cc0a9945c2ddf585684beb39ac@40.68.248.100:30303"
	]

https://github.com/paritytech/parity/blob/594716dc1ec822feb0230c4d57e99c4810e4b813/ethcore/res/ethereum/kovan.json#L66

Also, are you interested to add native support for Sokol et al. in Parity?

**Comment by 5chdn** [comment]
Why do you have a different testnet for security testing?

**Comment by igorbarinov** [comment]
Hi Afri,
Thank you for mentioning this feature. At the moment, we have some dynamics in our enodes lists, e.g. we add nodes:
- on testnet we are testing new validators (10 atm) and HF (2 atm)
- on mainnet we add more independent bootnodes from the community (5 independent bootnode keepers)

Also, we decided to keep a list of enodes in a separate file to have spec.json file more solid. 
We will add our core bootnodes to spec.json.

> Also, are you interested to add native support for Sokol et al. in Parity?

We appreciate your idea to have native support of POA Core/Sokol in Parity client. We planned to submit a PR.

> Why do you have a different testnet for security testing?

Before the network launch, we hired an audit from a Security/DDoS team. Report: [ dsec.ru ](https://drive.google.com/file/d/1hIPGWLAFOKbmznACwa_fI3OG6jCHV2lB/view). They attacked the network. We created a short-term network for the RED team. This experience was very helpful and gave us some important security knowledge, e.g. we have a configuration for validators without open external ports. Thus, it's hard to DDoS them.
